Various Updates for Issue 133

mpitcel opened the pull request:
Closes #133 

Updates: 

- Slideout wider on smaller screens 
- Add space above Banner Heading 
- Make Details in Slideout change pointer for clickable area on hover 
- Add Close button to the bottom of the Slideout 
- Hide Cookie Banner and Slideout when printing 
- Mirror changes between the two HTML files 
- Changes for V3 instead of V2 
- Add missing `type` to `button` tags 
- Hover and Focus both change button appearance 
- Added popup to links in the Cookie Banner and the About Slideout 
- Added `active` options to the CSS in addition to `hover` and `focus` 
- Various reformatting for readability 
- Focus on the Cookie Banner on Load

mpitcel committed:
Changes:
- Slideout wider on smaller screens
- Add space above Banner Heading
- Make Details in Slideout change pointer for clickable area on hover
- Add Close button to the bottom of the Slideout
- Hide Cookie Banner and Slideout when printing
- Mirror changes between the two HTML files

mpitcel committed:
Updates for both files:
- Changes for V3 instead of V2
- Add missing `type` to `button` tags
- Hover and Focus both change button appearance
- Various reformatting for readability

mpitcel committed:
Changes:
- Added popup to links in the Cookie Banner and the About Slideout
- Added `active` options to the CSS in addition to `hover` and `focus`
-Various formatting updates

mpitcel committed:
Updates:
- Focus on the Cookie Banner on Load
- Change popup location in Slideout

mpitcel committed:
Modal-type updates:
- Add new `div` for modal-type shading behind the Cookie Banner when open
- Add JS functions to add and remove modal-type shading and enable/disable scrolling
- Add various CSS settings to work with new functionality

edthedev commented:
These are substantial improvements addressing feedback we have received. I'm inclined to get these merged, and start a new branch for the remaining planned updates.

Thanks @mpitcel!

mpitcel committed:
More updates:
- Addressed comments about alerting for opening slideover dialog
- Updated various wording per comments

mpitcel committed:
Disable modal and related functionality

edthedev commented on applications/ila-cookie-banner/ila-cookie-banner.html:
```diff
@@ -1,18 +1,40 @@
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="profile" href="https://gmpg.org/xfn/11">
     <title>ILA CookieB</title>
```
```suggestion
    <title>Cookie Banner Example</title>
```

edthedev commented:
Here's info about switching to CDN version 3 - https://builder3.toolkit.illinois.edu/getting_started/index.html

We gave it a quick try here, but removing v2 broke a bunch of things, so we will need to revisit.

edthedev committed:
Update applications/ila-cookie-banner/ila-cookie-banner.html
